link/src/test/java/com/stripe/android/link/ui/wallet/WalletViewModelTest.kt

```diff
@@ -115,34 +113,9 @@ class WalletViewModelTest {
     }
 
     @Test
-    fun `On initialization when initially selected item exists then it is selected`() = runTest {
-        val selected = PaymentDetailsFixtures.CONSUMER_PAYMENT_DETAILS.paymentDetails.first()
-        whenever(args.selectedPaymentDetails).thenReturn(
-            LinkPaymentDetails.Saved(
-                selected,
-                mock()
-            )
-        )
-        whenever(linkAccountManager.listPaymentDetails())
-            .thenReturn(Result.success(PaymentDetailsFixtures.CONSUMER_PAYMENT_DETAILS))
-
-        val viewModel = createViewModel()
-
-        assertThat(viewModel.initiallySelectedId).isEqualTo(selected.id)
-
-        verify(navigator, times(0)).navigateTo(anyOrNull(), anyOrNull())
-    }
-
-    @Test
-    fun `On initialization when initially selected is new then navigate to AddPaymentMethod`() =
+    fun `On initialization when prefilledCardParams is not null then navigate to AddPaymentMethod`() =
```

Comment:
There should be a test for when prefilledCardParams is not null?

Reply:
That's the test above, "On initialization when no payment details then navigate to AddPaymentMethod".

Reply:
On this test it is not null, on the test above it is null.